ang/crmMailing/Recipients.js

```diff
@@ -229,6 +229,11 @@
                 page_num: rcpAjaxState.page_i,
                 params: filterParams,
               };
+
+              if('civicrm_mailing' === rcpAjaxState.entity) {
+                params["api.MailingRecipients.getcount"] = {};
+              }
+
```

Comment:
Instead of the above if condition, can this be removed and adjusted in the above "case" section?  Something like -
    
    var filterParams = recipients = {}; //Initialize.
    .
    .
    case 'civicrm_mailing':
      filterParams = { is_hidden: 0, is_active: 1 };
      recipients["api.MailingRecipients.getcount"] = {}; //Add chain api param
      break;
    }
    var params = {
      input: input,
      page_num: rcpAjaxState.page_i,
      params: filterParams,
    };
    $.extend(params, recipients); //Merge to the main param